### PR TITLE
Risk score reasons

### PR DESCRIPTION
**Pull request by marselester**

### CODE REVIEW

File: lib/minfraud/model/factors.rb [hunk 1]
```diff
@@ -2,11 +2,22 @@
...
 require 'minfraud/model/insights'
 require 'minfraud/model/subscores'
+require 'minfraud/model/risk_score_reason'
 
 module Minfraud
   module Model
     # Model representing the Factors response.
     class Factors < Insights
+      # This field contains RiskScoreReason objects that describe
+      # risk score reasons for a given transaction
```
Comment (ugexe) [inline lib/minfraud/model/factors.rb:12]: Spacing / line break is off

File: lib/minfraud/model/risk_score_reason.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      # This field contains Risk objects that describe one of the reasons for the multiplier.
+      #
+      # @return [Array<Minfraud::Model::Risk>]
+      attr_reader :reasons
+
+      # @!visibility private
+      def initialize(record)
+        super
+
+        if record
```
Comment (ugexe) [inline lib/minfraud/model/risk_score_reason.rb:26]: Could this just be `@multiplier = get('multiplier')`?

**Review by ugexe — CHANGES_REQUESTED** [review]

File: minfraud.gemspec [hunk 3]
```diff
@@ -22,10 +22,10 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'connection_pool', '~> 2.2'
-  spec.add_runtime_dependency 'http', '>= 4.3', '< 6.0'
-  spec.add_runtime_dependency 'maxmind-geoip2', '~> 1.2'
-  spec.add_runtime_dependency 'simpleidn', '~> 0.1', '>= 0.1.1'
+  spec.add_dependency 'connection_pool', '~> 2.2'
```
Comment (ugexe) [inline minfraud.gemspec:25]: I thought this was originally in its own commit

**Review by ugexe — APPROVED** [review]

**Comment by dan-jensen** [comment]
Thanks for adding this feature! Been using it in production for a week now, and it's working nicely 👍

**Pull request merged**